buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -54,15 +54,14 @@ object Versions {
     internal const val KEYBOARD_VISIBILITY_EVENT = "2.3.0"
     internal const val KFIXTURE = "0.2.0"
     internal const val KLUENT = "1.68"
-    internal const val KOTLIN = "1.5.31"
+    internal const val KOTLIN = "1.6.10"
     internal const val KOTLIN_BINARY_VALIDATOR = "0.8.0"
     internal const val LEAK_CANARY = "2.4"
     internal const val MATERIAL_COMPONENTS = "1.4.0"
     internal const val MARKWON = "4.6.2"
     internal const val MOCKITO_KOTLIN = "2.2.0"
     internal const val MOCKITO = "3.11.2"
-    // Be careful with updating Moshi to 1.13.0 as it's shipped as MR-jar which causes problems with Jetifier. Check [this](https://github.com/square/moshi/issues/1424) issue for more details
-    internal const val MOSHI = "1.12.0"
+    internal const val MOSHI = "1.13.0"
```

Comment:
I remember we reverted the KSP PR (#2793) before, because of Moshi `1.13.0` and I'm wondering if the same issue is not caused.

Reply:
It seems the problem was fixed in AGP `7.1.0`. Check [here](https://issuetracker.google.com/issues/159151549) and [here](https://issuetracker.google.com/issues/172784894)

Reply:
Yeah the only issue here is that some of our customers will have to migrate to 7.x.x AGP and they're using 4.x.x at the moment.

But honestly, stability wise and related to everything in all libraries around in the android world, they should update anyway.